src/composables/useDayjsHelper.ts

```diff
@@ -1,32 +1,45 @@
-import dayjs from "dayjs"
+import dayjs, { Dayjs } from "dayjs"
 import { computed } from "vue"
 
 import type { GGanttChartConfig } from "../components/GGanttChart.vue"
 import type { GanttBarObject } from "../types"
 import provideConfig from "../provider/provideConfig.js"
 
+export const DEFAULT_DATE_FORMAT = "YYYY-MM-DD HH:mm"
+
 export default function useDayjsHelper(config: GGanttChartConfig = provideConfig()) {
   const { chartStart, chartEnd, barStart, barEnd, dateFormat } = config
 
   const chartStartDayjs = computed(() => toDayjs(chartStart.value))
   const chartEndDayjs = computed(() => toDayjs(chartEnd.value))
 
-  const toDayjs = (value: string | GanttBarObject, startOrEnd?: "start" | "end") => {
-    if (typeof value === "string") {
-      return dayjs(value, dateFormat.value, true)
+  const toDayjs = (input: string | Date | GanttBarObject, startOrEnd?: "start" | "end") => {
+    let value
+    if (startOrEnd !== undefined && typeof input !== "string" && !(input instanceof Date)) {
+      value = startOrEnd === "start" ? input[barStart.value] : input[barEnd.value]
     }
-    if (startOrEnd == null) {
-      throw Error(
-        "VueGanttastic - toDayjs: passed a GanttBarObject as value, but did not provide start|end parameter."
-      )
+    if (typeof input === "string") {
+      value = input
+    } else if (input instanceof Date) {
+      return dayjs(input)
     }
-    const property = startOrEnd === "start" ? value[barStart.value] : value[barEnd.value]
-    return dayjs(property, dateFormat.value, true)
+    const format = dateFormat.value || DEFAULT_DATE_FORMAT
+    return dayjs(value, format, true)
+  }
+
+  const format = (input: string | Date | Dayjs, template?: string | false | undefined) => {
```

Comment:
This is a slim wrapper around the dayjs format function.
If the template value is false it will return a js date object though.